src/Raven.Studio/typescript/viewmodels/database/query/query.ts

```diff
@@ -1382,20 +1389,19 @@ class query extends viewModelBase {
         eventsCollector.default.reportEvent("query", "delete-documents");
         // Run the query so that we have an idea of what we'll be deleting.
         this.runQuery();
-        this.queryFetcher()(0, 1)
-            .done((results) => {
-                if (results.totalResultCount === 0) {
-                    app.showBootstrapMessage("There are no documents matching your query.", "Nothing to do");
-                } else {
-                    this.promptDeleteDocsMatchingQuery(results.totalResultCount);
-                }
-            });
+        
```

Comment:
do we need runQuery call if we don't call queryFetcher?

Reply:
refactored...